fintech-examples/fintech-api/src/main/resources/static/fintech_api.yml

```diff
@@ -416,6 +416,28 @@ paths:
         "401":
           $ref: "#/components/responses/401_Unauthorized"
 
+  /v1/consent/{userid}/{password}:
```

Comment:
Can we make password to be in request header? While the controller is for debug, still we can send password properly

Reply:
it should be called by browser, not http client. we can remove password completly.